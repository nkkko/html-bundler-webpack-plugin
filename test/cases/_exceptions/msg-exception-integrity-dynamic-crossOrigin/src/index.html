<!DOCTYPE html>
<html>
<head>
  <title>Test</title>
  <!-- load source script -->
  <script src="./main.js" defer="defer"></script>
</head>
<body>
  <h1>Hello World!</h1>
</body>
</html>