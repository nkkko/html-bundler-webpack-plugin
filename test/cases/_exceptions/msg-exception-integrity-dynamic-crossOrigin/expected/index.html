<!DOCTYPE html>
<html>
<head>
  <title>Test</title>
  <!-- load source script -->
  <script src="main.4461fe16.js" defer="defer" integrity="sha384-AsYjeunkery5Po1QSuEpFp4VwE/NrxSsVSaJgOVzvud37w+BgsR4Z8+QRgllbS0i" crossorigin="anonymous"></script>
</head>
<body>
  <h1>Hello World!</h1>
</body>
</html>